android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         buildToolsVersion = "35.0.0"
         minSdkVersion = 24
         compileSdkVersion = 35
-        targetSdkVersion = 35
+        targetSdkVersion = 34
```

Comment:
intentional downgrade?

Reply:
Yeah, this wasnt supposed to happen until react native 78 and we are waiting on expo to update